### PR TITLE
Travis: print which files are processed

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

**Comment by Sumith1896** [comment]
Cool. +1

**Comment by CodeMaxx** [comment]
+1

**Comment by Sumith1896** [comment]
@certik I don't see the result here https://travis-ci.org/symengine/symengine/jobs/122611136 
What's wrong?

**Comment by certik** [comment]
@Sumith1896 that's why I added it there, because I suspect it is not being run... No idea what's wrong.

**Comment by certik** [comment]
And there are failures, as I suspected...

File: bin/travis_clang.sh [hunk 1]
```diff
@@ -13,7 +18,11 @@ if [[ "${TRAVIS_OS_NAME}" == "linux" ]] && \
 
     FILES=`git ls-files | grep -E "\.(cpp|h|hpp)$" | grep -Ev "symengine/utilities" | grep -Ev "cmake/"`
 
+    echo "FILES="
+    echo $FILES
```
Comment (Sumith1896) [inline bin/travis_clang.sh]: Are we interested in printing this out?

**Comment by Sumith1896** [comment]
Awesome! It now works as expected. 
We can merge it once tests pass.

File: bin/travis_clang.sh (hunk 1 above)
Comment (certik) [inline bin/travis_clang.sh]: No, we don't need it, I removed it in 07828975d5030d71f318db182459ceaf9cdffb52.

**Comment by certik** [comment]
The fix in 3ce37c55f25c39b47c988d9cd4ef73cbbafa36e4 should hopefully prevent such failures in the future. I don't know why the extra checks where there in the first place.

**Comment by certik** [comment]
Tests pass, so I am going to merge this.

**Pull request merged**